File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.Ex_PB-JS-Nested-Conditional-Statements/P01_Cinema.js
function cinema(input){
    let cinemaType=input[0];
    let r=Number(input[1]);
    let c=Number(input[2]);

    switch(cinemaType){
        case "Premiere":
            console.log(`${(r*c*12).toFixed(2)} leva`);
            break;
        case "Normal":
            console.log(`${(r*c*7.5).toFixed(2)} leva`);
            break;
        case "Discount":
            console.log(`${(r*c*5).toFixed(2)} leva`);
            break;
    }
}

cinema(["Discount",
"12",
"30"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03Ex_ConditionalStatements_Advanced/P01_Cinema.java
package P03Ex_ConditionalStatements_Advanced;

import java.util.Scanner;

public class P01_Cinema {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String cinemaType = scanner.nextLine();

        int rows = Integer.parseInt(scanner.nextLine());
        int columns = Integer.parseInt(scanner.nextLine());

        switch (cinemaType) {
            case "Premiere":
                System.out.printf("%.2f leva", rows * columns * 12.0);
                break;
            case "Normal":
                System.out.printf("%.2f leva", rows * columns * 7.5);
                break;
            case "Discount":
                System.out.printf("%.2f leva", rows * columns * 5.0);
                break;
        }
    }
}
